docs/reference/requirement-sets/onenote-api-requirement-sets.md

```diff
@@ -16,10 +16,6 @@ The following table lists the OneNote requirement sets, the Office host applicat
 |:-----|:-----|
 | [OneNoteApi 1.1](/javascript/api/onenote?view=onenote-js-1.1)  | September 2016 |  
 
-## Office Common API requirement sets
-
-For information about Common API requirement sets, see [Office Common API requirement sets](office-add-in-requirement-sets.md).
```

Comment:
Why was this taken out?

Reply:
@AlexJerabek It's included - see the bottom of the page. I can also keep this one and remove the later one.

Reply:
No, this one seems like the better removal candidate. Thanks!